app/javascript/components/explore/ExploreDisplayTabs.jsx

```diff
@@ -176,26 +176,31 @@ function getFacetsParam(initFacets, selection) {
   const minimalSelection = {}
 
   const initSelection = {}
-  initFacets.filter(f => !f.isSelectedAnnotation).forEach(facet => {
+  initFacets.filter(f => !f.isSelectedAnnotation)?.forEach(facet => {
```

Comment:
Just curious - what does the "?" do?

Reply:
`?.` is the [optional chaining operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining), which:

> accesses an object's property or calls a function. If the object accessed or function called using this operator is undefined or null, the expression short circuits and evaluates to undefined instead of throwing an error.

At a high level, `?` is sometimes a useful quick fix for a UI-breaking error of trying to access a property in a non-existent object.  However, due to it being somewhat opaque and suppressing exceptional states, IMO it's best to use sparingly.